spec/workers/pull_request_monitor/pr_branch_record_spec.rb

```diff
@@ -0,0 +1,106 @@
+require "spec_helper"
+
+RSpec.describe PullRequestMonitor::PrBranchRecord do
+  describe ".create" do
+    it "has git create a local pr branch" do
+      repo = spy("CommitMonitorRepo")
```

Comment:
it has a distinct purpose which i find quite useful in tests like these where i'm trying to keep the setup phase under control. it's basically syntactic sugar for `double("CommitMonitorRepo").as_null_object`. i tend to use this quite a lot when creating doubles for collaborators whose use are just noisy implementation details which are not relevant to the particular test at hand. any thoughts?